### DISCUSSION TITLE
How often is Task-views updated?

### DISCUSSION THREAD

**Discussion by serkor1** [flaming-codes/crane-app#173] [Q&A] [answered]
Hi,

I am still a huge fan of your app. All my links are redirecting towards your page.

Out of curiorsity; how often, if at all, does your app update the CRAN Task Views under the documentation?

I have a package which got listed there, some 48 hours(ish) ago - but this is not visible on your App as of now.

Best,

**Comment by lschoenm** [comment, marked as answer]
Hello @serkor1, great to hear that you like our app! 

Did your package [cryptoQuotes](https://www.cran-e.com/package/cryptoQuotes) get added to the task view after an updated version had been published to CRAN, i.e., is this event separated from your package appearing on the CRAN "Packages By Date of Publication" list? 

Currently, we are updating our data based on this list, with our scraper checking every three hours whether new packages have been added to the list. Only then is a package's metadata newly scraped and updated on CRAN/E. Checking over 21,000 packages regularly for updated task views would probably be too demanding for our present setup. 

Best, 
Lukas

**Reply by serkor1** [reply]
Hi @SpaceCowboy-71,

Yes - I've added the package to the Task View after CRAN publication. 

I wouldn't even start to pretend that I have the sligthest knowledge of the kind of work you do; so I can't even suggest anything 🤷‍♀️😂

It was mostly out of curiorsity!

Best,

**Reply by lschoenm** [reply]
Ah, so that explains why it is not updated yet. For now, I fear you will have to wait until you push another update of your package to CRAN for the task view to be displayed on our site. 

Nevertheless, thank you for making us aware of this issue. Maybe we will implement a simple solution - actually it shouldn't be too much trouble!

Best, 
Lukas
